configuration/examples/dev/main.jsonnet

```diff
@@ -136,8 +138,38 @@ dev.manifests +
 {
   'minio-deployment': minio.deployment,
   'minio-pvc': minio.pvc,
-  'minio-secret-thanos': minio.secretThanos,
-  'minio-secret-loki': minio.secretLoki,
+  'minio-secret-thanos': {
+    apiVersion: 'v1',
+    kind: 'Secret',
+    metadata: {
+      name: 'thanos-objectstorage',
+      namespace: dev.config.namespace,
+    },
+    stringData: {
+      'thanos.yaml': |||
+        type: s3
+        config:
+          bucket: thanos
+          endpoint: %s.%s.svc.cluster.local:9000
+          insecure: true
+          access_key: minio
```

Comment:
just a nit: We can use variables for access_key and secret_key. They are referred more in a couple of places.

Reply:
yeah I wanted to keep the changes minimal and improve bit by bit, it's already better than it was before and I think this can be done as follow ups